### ISSUE TITLE
I ran npx prisma migrate save --experimental and am getting error like this:   ERROR  Oops, an unexpected error occured! Error in migration engine.

### ISSUE THREAD

**Issue by kenule500** [prisma/migrate#455]

**Comment by pantharshit00** [comment]
Can you please share your schema and paste the output of your terminal here? 

Please take some time and follow the issue template :pray:, I will suggest you close this issue and open a new one in which you have filled all of the required information: https://github.com/prisma/migrate/issues/new?assignees=&labels=&template=bug_report.md&title=

Otherwise, please provide enough context here so that we can reproduce this.

**Comment by floydianslips** [comment]
I am getting this same error. Hopefully this info will help....?

## Bug description
Error in Migration engine. Upon running `yarn rw db save` I received this error: 

```
Error in migration engine.
Reason: [migration-engine/connectors/sql-migration-connector/src/lib.rs:240:9] Deserializing the database migration fai
led.: Error("missing field `dataType`", line: 0, column: 0)

```

## How to reproduce
From terminal run `yarn rw db save`

## Expected behavior
This is the result of the first time I ran `yarn rw db save`:

```
vagrant% yarn rw db save
yarn run v1.22.4
$ /vagrant/redwood-test/node_modules/.bin/rw db save
Creating database migration... [started]
$ /vagrant/redwood-test/node_modules/.bin/prisma migrate save --experimental
✔ Name of migration … create posts
📼  migrate save --name create posts

New datamodel:

datasource DS {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}

generator client {
  provider      = "prisma-client-js"
  binaryTargets = env("BINARY_TARGET")
}

model Post {
  id        Int      @default(autoincrement()) @id
  title     String
  body      String
  createdAt DateTime @default(now())
}


Prisma Migrate just created your migration 20200519190508-create-posts in

migrations/
  └─ 20200519190508-create-posts/
    └─ steps.json
    └─ schema.prisma
    └─ README.md

Run prisma migrate up --experimental to apply the migration

Creating database migration... [completed]
Done in 41.50s.
```


## Prisma information
```
datasource DS {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}

generator client {
  provider      = "prisma-client-js"
  binaryTargets = env("BINARY_TARGET")
}

model Post {
  id        Int      @default(autoincrement()) @id
  title     String
  body      String
  createdAt DateTime @default(now())
}

model Message {
  id        Int      @default(autoincrement()) @id
  name      String
  email     String
  message   String
  createdAt DateTime @default(now())
}
```

## Environment & setup
Following RedwoodJS Tutorial Part 3 (https://www.youtube.com/watch?time_continue=516&v=eT7iIy0F8Tk&feature=emb_logo)
- OS: Debian 10
- Database: sqlite
- Prisma Version: Prisma CLI version: prisma/1.34.10 (linux-x64) node-v12.13.0
- Vagrant Version: 2.2.3
- Virtualbox Version: 6.1.8 r137981 (Qt5.11.3)

**Comment by pantharshit00** [comment]
Looks like the migration are out of sync. Migrate is in experimental phase so this might happen sometimes.

In order to fix it you will need to reset your migrations. Delete the `_Migration` table in your database and also delete `prisma/migrations` folder and try again with the latest version

Docs: https://prisma.io/docs/reference/tools-and-interfaces/prisma-migrate#troubleshooting

**Comment by homerjam** [comment]
I get this issue even after completing the troubleshooting steps. However there's no clue as to what is happening even with the `--verbose` flag. I have submitted the error report but why is it that the errors aren't shown in the terminal or a log is generated somewhere?

**Comment by floydianslips** [comment]
> Looks like the migration are out of sync. Migrate is in experimental phase so this might happen sometimes.
> 
> In order to fix it you will need to reset your migrations. Delete the `_Migration` table in your database and also delete `prisma/migrations` folder and try again with the latest version
> 
> Docs: https://prisma.io/docs/reference/tools-and-interfaces/prisma-migrate#troubleshooting

This seems to have solved the issue I was having. I went through the entire Redwood tutorial again and didn't have the same issue either. I did upgrade Prisma the first time before the error so perhaps that led to the issue...?

**Comment by pantharshit00** [comment]
@floydianslips Yes, right now Prisma migrate is experimental so format change can happen which can cause this.

@homerjam Try running it in debug mode:
```
export DEBUG=*
# commands
```

**Comment by tomhoule** [comment]
This can be worked around by deleting the `_Migration` table and the migrations folder.

**Comment by tomhoule** [comment]
Thanks for taking the time to report this issue!

We've recently released a Prisma Migrate Preview ([2.1.3.0 release notes](https://github.com/prisma/prisma/releases/tag/2.13.0)), which has quite a few changes compared to the previous experimental version. We believe this issue is no longer relevant in this new version, so we are closing this.

We would encourage you to try out Prisma Migrate Preview. If you come across this or any other issue in the preview version, please feel free to open a [new issue in prisma/prisma](https://github.com/prisma/prisma/issues/new/choose).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue](https://github.com/prisma/prisma/issues/4531).
